### DISCUSSION TITLE
Trigger setCustomValidity instead of auto correction

### DISCUSSION THREAD

**Discussion by corny** [dm4t2/vue-currency-input#339] [Ideas]
Auto correction could lead to submission of unwanted inputs if the user is not aware of it.
I would like to see an error message (i.e. using `setCustomValidity`) to enforce the user to correct the input.

**Comment by dm4t2** [comment]
I would suggest using a custom form validation instead of the built-in value range validation.

Example: https://codesandbox.io/s/using-vue-currency-input-with-quasar-veevalidate-gokw6?from-embed=&file=/src/QCurrencyInput.vue
